src/analysis/retail/evoker/augmentation/modules/breahtofeons/BreathOfEonsHelper.tsx

```diff
@@ -0,0 +1,538 @@
+            color: '#736F4E',
+            label: 'Current Breath timing',
+            strokeWidth: 5,
+          },
+          {
+            spellTracker: [
+              {
+                timestamp: topWindow.start,
+                count: 1 * (topWindow.sum / damageInRange),
+              },
+              {
+                timestamp: topWindow.end,
+                count: 0,
+              },
+            ],
+            type: 'area',
+            color: '#4C78A8',
+            label: 'Optimal Breath timing',
+            strokeWidth: 5,
+          },
+        ];
+
+    const newGraphData = generateGraphData(
+      dataSeries,
+      breathStart - buffer,
+      breathEnd + buffer,
+      'Breath Window',
+      !topWindow ? <>You didn't hit anything.</> : undefined,
+    );
+
+    return newGraphData;
+  }
+
+  function generateExplanationContent(
+    topWindow: any,
+    inRangeSum: any,
+    damageToDisplay: number,
+    damageInRange: number,
+    lostDamage: number,
+    earlyDeadMobsDamage: number,
+  ) {
+    if (!topWindow) {
+      return <div></div>;
+    }
+
+    const damageSourcesOptimal = [];
+    /** These specific colors are choosen because certain colors seems to completely break the donut
+     * And randomizes the ouput ie. values are not properly orded. */
+    const colorMap = ['#2D3142', '#4F5D75', '#BFC0C0', '#EF8354', '#FFFFFF'];
+
+    for (let i = 0; i < topWindow.sumSources.length; i += 1) {
+      const source = topWindow.sumSources[i];
+      const playerInfo = playerNameMap.get(source.sourceID);
+      damageSourcesOptimal.push({
+        color: colorMap[i],
+        label: playerInfo?.name,
+        valueTooltip: formatNumber(source.damage * BREATH_OF_EONS_MULTIPLIER),
+        value: source.damage,
+      });
+    }
+
+    const damageSourcesCurrent = [];
+    /** These specific colors are choosen because certain colors seems to completely break the donut
+     * And randomizes the ouput ie. values are not properly orded. */
+
+    for (let i = 0; i < inRangeSum.length; i += 1) {
+      const source = inRangeSum[i];
+      const playerInfo = playerNameMap.get(source.sourceID);
+      damageSourcesCurrent.push({
+        color: colorMap[i],
+        label: playerInfo?.name,
+        valueTooltip: formatNumber(source.damage * BREATH_OF_EONS_MULTIPLIER),
+        value: source.damage,
+      });
+    }
+
+    const content: JSX.Element = (
+      <div className="flex-container">
+        <div className="table">
+          <div className="flex-row">
+            <div className="flex-cell">
+              <TooltipElement
+                content="Because of the way Blizzard handles damage attribution, the values 
+                    displayed here may have a small margin of error. Additionally, if an enemy 
+                    becomes immune or takes reduced damage when your Breath of Eons explodes, this 
+                    value may also be overestimated, e.g. Neltharion going immune mid-Breath."
```

Comment:
as useful as it currently is, id remove the example as soon enough people wont be familiar with the encounter anymore